Makefile

```diff
@@ -295,6 +295,10 @@ else
 $(error Error: unsupported option "debug=$(debug)". Options are 'full', 'none', or 'base')
 endif
 
+## Sixth, parse CPU local sections.
+	@echo -e "Parsing CPU local sections"
```

Comment:
nit: are we parsing them, or modifying/fixing up the ELF sections? Use whatever terms you think are most accurate.